ledger/src/blockstore_db.rs

```diff
@@ -35,6 +35,8 @@ use {
     thiserror::Error,
 };
 
+pub const DEFAULT_ROCKS_FIFO_SHRED_STORAGE_SIZE_BYTES: u64 = 250 * 1024 * 1024 * 1024;
```

Comment:
Can you please add a comment that provides some context regarding this magic number.